modules/swagger-codegen/src/main/resources/go/api.mustache

```diff
@@ -5,7 +5,7 @@ import (
     "strings"
     "fmt"
     "encoding/json"
-    "errors"
+    "reflect"
```

Comment:
@wing328  there is a potential issue with this reference, when there is no model in the parameter(line 65 generate the required parameter validation), if we find a better way to compare null object then we don't need this reference.